tools/camera_macro/macros.xml

```diff
@@ -165,4 +165,23 @@
             <citation type="doi">10.1093/bioinformatics/btu813</citation>
         </citations>
     </xml>
+    <xml name="edam_topics">
+        <edam_topics>
+            <edam_topic>topic_2259</edam_topic> <!-- Systems biology -->
+            <edam_topic>topic_3172</edam_topic> <!-- Metabolomics -->
+            <edam_topic>topic_3307</edam_topic> <!-- Computational biology -->
```

Comment:
Any of the other term are children of `Computational Biology` !?
But I don't know if I can pick a more precise term in `Computational Biology` ?
If not, I can understand that it should be remove because not enough specific